tenzir/integration/tests/setup_suite.bash

```diff
@@ -19,5 +19,6 @@ if ! command -v tenzir; then
 fi
 
 libpath_relative_to_binary="$(realpath "$(dirname "$(command -v tenzir)")")/../share/tenzir/integration/lib"
-libpath_relative_to_pwd="${BATS_TEST_DATADIR%%/integration/*}/lib"
+libpath_relative_to_pwd="${BATS_TEST_DIRNAME%%/integration/*}/integration/lib"
```

Comment:
I don't understand this change, can you explain? Wasn't this something we needed to copy over to every single plugin?

Reply:
`BATS_TEST_DATADIR` straight up doesn't exist, the other plugins all already have the correct `BATS_TEST_DIRNAME` so I think it was just an oversight here that was just never noticed because the other path already contained the bats libraries.

Reply:
Ah, that makes sense. Thanks!